Product Designer/assets/css/styles.css

```diff
@@ -0,0 +1,203 @@
+/* UNIVERSAL STYLES */
+*{
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    color: white;
+    padding: 0;
+    box-sizing: border-box;
+    margin: 0px;
+}
+body{
+    background-color: black;
+}
+
+/* MAIL SECTION STYLES*/
+.mailSection{
+    width: 74%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: end;
+    font-size: 2.5em;
+    font-weight: 900;
```

Comment:
What if the font you're using doesn't have a font-weight of 900, how will this work ?

Reply:
Now as i have imported the fonts along with 900 weight this is now working fine